src/components/DatePicker/transform-value-to-date-or-range.ts

```diff
@@ -99,6 +99,10 @@ export const transformRelativeDateRangeToDateRange = (
   return { start, end };
 };
 
+export function setDateToMidnight(value: Date): Date {
+  return new Date(new Date(value).setUTCHours(0, 0, 0, 0));
+}
+
 export const setDateRangeHours = (value: DateRange | string | undefined): DateRange | undefined => {
   if (!isDateRange(value) || !value.start || !value.end) return;
   const startHours = { hour: 0, minute: 0, second: 0, millisecond: 0 };
```

Comment:
note: on devrait peut être renommer ce fichier j'ai l'impression qu'il ne fait pas uniquement de la transformation de valeur du coup :wink:

Reply:
I'm gonna remove this function and use `ŧimeAdjust`

Reply:
2cd56bb